src/pages/Landing/components/StudentTicket/History/History.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-nested-ternary */
+import React from 'react'
+import styles from './History.module.scss'
+import head from './mockData'
+import deleteIcon from '../../../../../components/icons/icons-svg/delete.svg'
+
+function History() {
+  const Priorities={
```

Comment:
should move enums to `src/enums` and don't declare enums inside the component, It will be re-declared when the component changed.